proposals/NNNN-Adding-Audible-Status-Notification.md

```diff
@@ -0,0 +1,66 @@
+# Adding Audible Status Notification
+
+* Proposal: [SDL-NNNN](NNNN-Adding-Audible-Status-Notification.md)
+* Author: [Shohei Kawano](https://github.com/Shohei-Kawano)
+* Status: **Create**
+* Impacted Platforms: [Core / iOS / Java Suite / RPC /HMI]
+
+## Introduction
+
+Currenlty, there is no way to independently notify the Audible status to applications that output audio such as SDL Media.  
+This time, `OnAudibleStatus` is added as a new API so that the status of Audible can be notified.
+
+
+## Motivation
+
+As a means to notify the Audible status to applications that output audio.  
+In the case of Mobile from Core, Audible without changing the HMI Level by using the parameters `audioStreamingState` and hmiLevel in RPC `OnHMIStatus` Can be notified.  
```

Comment:
I'm not sure what you mean by "of Mobile from Core" - can you please clarify?

Reply:
It mean "Notification from Mobiel to Core in MOBILE_API"
Is this clarify?

Reply:
Are you suggesting that mobile requests an audio state, or do you want to use the hmi to update the audio state without changing the hmi level?

Reply:
If we understand your request correctly, we'd suggest revising to the following:
```suggestion
Currently, the `OnHMIStatus` notification from Mobile to Core can set an Audible state without changing the hmiLevel by using the parameters `audioStreamingState` and hmiLevel
```